integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -0,0 +1,66 @@
+import typing as tp
+import allure
+import json
+from jsonschema import Draft4Validator
+import pytest
```

Comment:
please run black to reformat imports (python internal libraries, 3rd party, self package)